bdb/bdb_int.h

```diff
@@ -1166,7 +1166,7 @@ enum {
 extern pthread_key_t bdb_key;
 
 char *bdb_strerror(int error);
-char *bdb_trans(const char infile[], char outfile[]);
+char *bdb_trans(const char infile[PATH_MAX], char outfile[PATH_MAX]);
```

Comment:
I made this take a size_t outsz in a different branch (and changed callers) (and changed callers).